### PR TITLE
doc: line-edit PendingReleaseNotes

### PR DESCRIPTION
**Pull request by zdover23**
Edit PendingReleaseNotes for clarity prior to Reef RCs.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: PendingReleaseNotes
Comment (jdurgin) [inline PendingReleaseNotes:82]: This fix has already been backported to pacific and quincy, is it still worth mentioning here @rzarzynski @neha-ojha ?

**Review by jdurgin — APPROVED** [review]
Good wording and clarity improvements!

We'll want to reorganize (sort by component, importance) and add more general release notes before the final release, e.g. what we did for quincy in https://github.com/ceph/ceph/pull/45048

**Comment by zdover23** [comment]
> Good wording and clarity improvements!
> 
> We'll want to reorganize (sort by component, importance) and add more general release notes before the final release, e.g. what we did for quincy in #45048

Thanks!

We'll wait until your final sign-off before we merge this.

**Pull request closed without merging**
